phpunit.xml.dist

```diff
@@ -7,7 +7,7 @@
     <env name="SIMPLETEST_BASE_URL" value="${drupal.base_url}"/>
     <env name="SIMPLETEST_DB" value="mysql://${drupal.database.user}:${drupal.database.password}@${drupal.database.host}:${drupal.database.port}/${drupal.database.name}"/>
     <env name="SIMPLETEST_SPARQL_DB" value="sparql://${drupal.sparql.host}:${drupal.sparql.port}/?module=sparql_entity_storage"/>
-    <env name="MINK_DRIVER_ARGS_WEBDRIVER" value='["${selenium.browser}", null, "${selenium.host}:${selenium.port}/wd/hub"]'/>
+    <env name="MINK_DRIVER_ARGS_WEBDRIVER" value='["${selenium.browser}", {"browserName":"chrome","chromeOptions":{"w3c": false, "args":["--no-sandbox", "--start-maximized", "--disable-gpu", "--window-size=1440,900", "--disable-dev-shm-usage", "--disable-setuid-sandbox", "--disable-web-security", "--DNS-prefetch-disable", "--disable-translate", "--ignore-certificate-errors", "--test-type", "--disable-extensions", "--incognito", "--disable-infobars"]}}, "${selenium.host}:${selenium.port}/wd/hub"]'/>
```

Comment:
Window size here should reflect the old one and be 1280x800.

Reply:
We also need to bring these to behat.yml.dist .
I'm copying here the ones from joinup, update them based on the ones used above:
```
capabilities:
          browser: chrome
          nativeEvents: true
          marionette: true
          browserName: chrome
          version: '*'
          extra_capabilities:
            chromeOptions:
              w3c: false
              args:
                - '--no-sandbox'
                - '--start-maximized'
                - '--disable-gpu'
                - '--window-size=1600,1200'
                - '--disable-dev-shm-usage'
                - '--disable-setuid-sandbox'
                - '--disable-web-security'
                - '--DNS-prefetch-disable'
                - '--disable-translate'
                - '--ignore-certificate-errors'
                - '--test-type'
                - '--disable-extensions'
                - '--incognito'
                - '--disable-infobars'
                - '--allowed-ips=*'
                - '--whitelisted-ips=*